apps/judicial-system/backend/src/app/modules/case/case.service.ts

```diff
@@ -1337,15 +1340,19 @@ export class CaseService {
           )?.subpoenas?.[0]?.id !== updatedDefendant.subpoenas?.[0]?.id,
       )
       .map((updatedDefendant) => [
-        {
-          type: MessageType.DELIVERY_TO_POLICE_SUBPOENA,
-          user,
-          caseId: theCase.id,
-          elementId: [
-            updatedDefendant.id,
-            updatedDefendant.subpoenas?.[0].id ?? '',
-          ],
-        },
+        ...(updatedCase.origin === CaseOrigin.LOKE
```

Comment:
The case must be created from LÖKE, otherwise the upload fails. This prevents us spamming LÖKE with unsuccessful calls.